acl-engine/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+  id("java-library")
+  id("configure-java")
+  id("groovy")
+}
+
+dependencies {
+  api projects.aclGroovyDsl
+  api projects.model
+  api libs.groovy.all
```

Comment:
I think if we started including Groovy in the main code base (not only test), we should declare only the necessary dependencies:
https://mvnrepository.com/artifact/org.apache.groovy